scripts/upgrade

```diff
@@ -94,12 +94,13 @@ fi
 #=================================================
 # INCREASE INOTIFY'S LIMITS
 #=================================================
-ynh_script_progression --message="Increase inotify's limits" --weight=2
+ynh_script_progression --message="Increasing inotify's limits..." --weight=2
 
 # Increase the maximum number of files inotify can monitor.
 cp -a ../conf/90-inotify_minidlna.conf /etc/sysctl.d/
 # Then, reload the kernel configuration.
-if ! IS_PACKAGE_CHECK; then
+if ! IS_PACKAGE_CHECK   # LXC doesn't allow sysctl to play with kernel options.
```

Comment:
Dunno if that's relevant but we had a discussion in transmission a while ago about the usage of this technique ... sure that fixes tests, but that also means that the app won't install on regular LXC instances (for example, our very own infra is based on yunohost LXCs ...) which are a legit use case. One could use systemctl-detect-virt to detect that we're inside a lxc (or idk which trick we ended up using in transmission)

Anyway, I understand that the point of the PR was only to changes the syntax + add the comment :/